src/Sylius/Bundle/ApiBundle/Resources/config/services/context_builders.xml

```diff
@@ -43,5 +70,14 @@
         >
             <argument type="service" id=".inner" />
         </service>
+
+        <service
```

Comment:
Would be cool to place this service and `SerializerContextBuilder\LoggedInCustomerEmailIfNotSetAwareContextBuilder` one after another.